.github/workflows/tests.yml

```diff
@@ -65,6 +65,26 @@ jobs:
       - name: "Run the tests"
         run: /build/cpp/static/bin/tests
   
+  wasm:
```

Comment:
This just checks that the wasm compiles. We'll still check its execution in the faasm tests